README.md

```diff
@@ -50,28 +50,27 @@ For provisioning the following tools will be used:
-|--------------------------------------------------------------------|---------------------------------------------------------------------|
-| [ansible](https://www.ansible.com)                                 | Preparing Ubuntu for Kubernetes and installing k3s                  |
-| [direnv](https://github.com/direnv/direnv)                         | Exports env vars based on present working directory                 |
-| [flux](https://toolkit.fluxcd.io/)                                 | Operator that manages your k8s cluster based on your Git repository |
-| [gnupg](https://gnupg.org/)                                        | Encrypts and signs your data                                        |
-| [go-task](https://github.com/go-task/task)                         | A task runner / simpler Make alternative written in Go              |
-| [ipcalc](http://jodies.de/ipcalc)                                  | Used to verify settings in the configure script                     |
-| [jq](https://stedolan.github.io/jq/)                               | Used to verify settings in the configure script                     |
-| [kubectl](https://kubernetes.io/docs/tasks/tools/)                 | Allows you to run commands against Kubernetes clusters              |
-| [pinentry](https://gnupg.org/related_software/pinentry/index.html) | Allows GnuPG to read passphrases and PIN numbers                    |
-| [sops](https://github.com/mozilla/sops)                            | Encrypts k8s secrets with GnuPG                                     |
-| [terraform](https://www.terraform.io)                              | Prepare a Cloudflare domain to be used with the cluster             |
+| Tool                                               | Purpose                                                                                                                                 |
+|----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| [ansible](https://www.ansible.com)                 | Preparing Ubuntu for Kubernetes and installing k3s                                                                                      |
+| [direnv](https://github.com/direnv/direnv)         | Exports env vars based on present working directory                                                                                     |
+| [flux](https://toolkit.fluxcd.io/)                 | Operator that manages your k8s cluster based on your Git repository                                                                     |
+| [age](https://github.com/FiloSottile/age)          | A simple, modern and secure encryption tool (and Go library) with small explicit keys, no config options, and UNIX-style composability. |
```

Comment:
You can use https://age-encryption.org/ as well (not sure which one you prefer)

Reply:
Looks like that just forwards to the github link, the Github should be fine then.